static/docs/commands-reference/checkout.md

```diff
@@ -194,9 +194,8 @@ Let's check the `model.pkl` entry in `train.dvc` again:
 
 ```yaml
 outs:
-  - cache: true
-    path: model.pkl
     md5: a66489653d1b6a8ba989799367b32c43
+        path: model.pkl
```

Comment:
I think the previous one was correct. Check this doc: https://dvc.org/doc/user-guide/dvc-file-format

Reply:
done